guide/english/html/block-and-inline-elements/index.md

```diff
@@ -3,7 +3,7 @@ title: Block and Inline Elements
 ---
 ## Block and Inline Elements
 
-Let us understand them using below examples:
+Let us understand them using the examples below:
```

Comment:
In general we want everything written in the second-person point of view.  To avoid all of this, I think the line could be removed.

Reply:
Good point. I have removed the line.